Include/httpClient/pal.h

```diff
@@ -37,9 +37,18 @@
 #endif 
 
 #define _HRESULTYPEDEF_(_sc) ((HRESULT)_sc)
-#define E_TIME_CRITICAL_THREAD           RPC_E_WRONG_THREAD
+
+#ifndef E_TIME_CRITICAL_THREAD
+#define E_TIME_CRITICAL_THREAD           _HRESULTYPEDEF_(0x800701A0L)
+#endif
+
+#ifndef E_NOT_SUPPORTED
 #define E_NOT_SUPPORTED                  _HRESULTYPEDEF_(0x80070032L)
-#define E_INSUFFICIENT_BUFFER            _HRESULTYPEDEF_(0x8007007AL)
+#endif
+
+#ifndef E_NOT_SUFFICIENT_BUFFER
```

Comment:
Update the non win32 section, right now it still defines E_INSUFFICIENT_BUFFER

Where does this name come from anyway? http://errors/ does not find it.

Reply:
It's defined in winerror.h. I found it already defined as E_NOT_SUFFICIENT_BUFFER, which is why the rename. Updated code.